content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,488 @@
+---
+layout: engineering-education
+status: publish
+published: true 
+url: /statistical-hypothesis-testing-python-implementation/
+title: Statistical Hypothesis Testing With Python Implementation
+description: This tutorial is a comprehensive explanation and implementation of statistical hypothesis testing methods such as t-test, chi-squared test & ANOVA in python, It is well known that most of the implementation of these techniques are always in R, and most Data Scientists and Analysts who use python always shy away from it.
+author: qoyum-olatunde-yusuf
+date: 2021-06-16T00:00:00-11:00
+topics: [Statistical Analysis, Hypothesis Testing, Data Science, Python ]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/statistical-hypothesis-testing-python-implementation/hero.png
+ alt: statistic hypothesis testing with python cover image
+ 
+---
+Hypothesis is the act of making observations and generating probing questions from the observations made in an attempt to come to a conclusion. Hypothesis testing is the test of assumptions on a population sample. It involves checking out whether a hypothesis should be accepted or not. In the world of work, hypothesis testing has been of great importance in making worthy decisions off mere observations. Billions of data are being generated on a daily basis around the globe and this has made the relevance of hypothesis testing a stronghold in making business decisions.
+<!--more-->
+
+### Understanding Hypothesis Testing
+
+There are two types of hypothesis testing, namely:
+1. Null Hypothesis
+2. Alternate Hypothesis
+
+**Null hypothesis** is the initial assumption about an event (also referred to as ground truth), whereas alternate hypothesis is an assumption that counters the initial assumption"
+
+In a scenario whereby a septuagenarian woman has a swollen stomach or baby bump, our initial assumption is going to be that 
+**"She is pregnant"**.
+
+while our alternate hypothesis will be
+**"She is not pregnant"**.
```

Comment:
This example needs a little bit of context. Add an external resource for septuagenarian.
Also, since alternate hypothesis is the null hypothesis negated, consider rewording "...has a swollen stomach or a baby bump..." appropriately.
Also, replace while with whereas.